### DISCUSSION TITLE
how to display notification on the bottom-left?

### DISCUSSION THREAD

**Discussion by gogamid** [folke/snacks.nvim#178] [Q&A] [answered]
how to display notification on the bottom-left?

**Comment by folke** [comment]
Check the notifier settings in the doc?

**Comment by gogamid** [comment]
hi @folke, actually i did, and I set to 
```lua
        margin = { top = 0, right = 0, bottom = 0, left = 1 },
        top_down = false,
```
, but the notifications show up on the bottom right, i cannot put them to bottom left

**Reply by folke** [reply, marked as answer]
ah right. I misread that. nope, not supported.

**Reply by gogamid** [reply]
no problem, alright :)
